disnake/user.py

```diff
@@ -111,6 +116,9 @@ def _update(self, data: Union[UserPayload, PartialUserPayload]) -> None:
         self._public_flags = data.get("public_flags", 0)
         self.bot = data.get("bot", False)
         self.system = data.get("system", False)
+        self.collectibles: Optional[Collectibles] = None
```

Comment:
This should also be added here:
https://github.com/DisnakeDev/disnake/blob/75a6e77018f9b17a6c089dfe900ca3689b4b435a/disnake/member.py#L281
https://github.com/DisnakeDev/disnake/blob/75a6e77018f9b17a6c089dfe900ca3689b4b435a/docs/api/widgets.rst#L47